<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientBanqueWindow</class>
 <widget class="QMainWindow" name="ClientBanqueWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distributeur Automatique de Billets</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Adresse :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAdresse">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Port :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPort">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>8888</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonConnexion">
            <property name="text">
             <string>Connexion</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Etat de la connexion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetEtat"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Message de la banque :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditMessageBanque"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string>Opérations envoyées à la banque</string>
      </property>
      <widget class="QPushButton" name="pushButtonNumero">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Numéro de compte</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditNumero">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Montant :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditMontant">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonSolde">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Solde</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonRetrait">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>50</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Retrait</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonDepot">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Dépot</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonEnvoyer">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Envoi</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
